git-ftp

```diff
@@ -1590,7 +1590,7 @@ do
 						REMOTE_PASSWD="$2"
 						shift
 					else
-						print_error_and_die "Too few arguments for option -p." "$ERROR_MISSING_ARGUMENTS"
+						print_error_and_die "Too few arguments for option -p. Maybe the manual will help: https://github.com/git-ftp/git-ftp/blob/master/man/git-ftp.1.md#passwords" "$ERROR_MISSING_ARGUMENTS"
```

Comment:
Hm, this is a good idea but I'm worried about directing people to the master documentation even if people have a really old version. If Git-ftp is properly installed, the official way of reading the manual is `man git-ftp`. But maybe I'm a bit too old school and we should consider the master documentation valid for all versions (with version hints in there)?

I don't know.

Reply:
I see what you mean. I actually use both. But I also have no problem removing the hint. The suggestion was made [here](https://github.com/git-ftp/git-ftp/issues/468#issuecomment-501638166).

Reply:
Or we could make the link dynamic and use the current version to generate it. 

```sh
https://github.com/git-ftp/git-ftp/blob/$VERSION/man/git-ftp.1.md
```
And the anchor will also exit since it will be introduced in the version it is used. If a older version is installed the error message would not appear. And even if the anchor does not exits it will work since it is ignored then.

The only problem could be unreleased version, but I think people who use such a version could deal with it.

Reply:
I'm happy with the link for now. None of the other solutions convince me. :wink:

Reply:
Too bad … 😉

The dynamic link with the version or the static one to master?

Reply:
master